docs/index.rst

```diff
@@ -0,0 +1,299 @@
+ADB Enhanced |Downloads| |PyPI version| |Build Status| |CircleCI|
+=================================================================
+
+.. figure:: docs/logo.png
+   :alt: Logo
+
+   Logo
+
+Swiss-army knife for Android testing and development. A command-line
+interface to trigger various scenarios like screen rotation, battery
+saver mode, data saver mode, doze mode, permission grant/revocation.
+
+Release announcement
+--------------------
+
+See `Release
+announcement <https://ashishb.net/tech/introducing-adb-enhanced-a-swiss-army-knife-for-android-development/>`__
+
+Installation
+------------
+
+``sudo pip3 install adb-enhanced``
+
+Note
+----
+
+1. ``sudo pip install adb-enhanced`` for python2 based install works as
+   well but I would recommend moving to python3 since I will deprecate
+   python2 support after Dec 31, 2018.
+2. If you don’t have sudo access or you are installing without sudo then
+   ``adbe`` might not be configured correctly in the path.
+3. To setup bash/z-sh auto-completion, execute
+   ``sudo pip3 install infi.docopt-completion && docopt-completion $(which adbe)``
+   after installing adb-enhanced.
+
+Examples
+--------
+
+Device configuration
+~~~~~~~~~~~~~~~~~~~~
+
+-  Turn doze mode on
+
+   ``adbe doze on``
+
+-  Turn mobile-data off
+
+   ``adbe mobile-data off``
+
+-  Turn on battery saver
+
+   ``adbe battery saver on``
+
+-  Don’t keep activities in the background
+
+   ``adbe dont-keep-activities on``
+
+-  Take a screenshot ``adbe screenshot ~/Downloads/screenshot1.png``
+
+-  Take a video ``adbe screenrecord video.mp4 # Press ^C when finished``
+
+Permisions
```

Comment:
Typo

Reply:
Thanks for pointing out. This is a typo in README.md as well. I am fixing that and some other issues in https://github.com/ashishb/adb-enhanced/pull/74